core/src/test/java/org/opensearch/sql/analysis/ExpressionAnalyzerTest.java

```diff
@@ -487,6 +487,15 @@ void simple_query_string_expression_two_fields() {
             AstDSL.unresolvedArg("query", stringLiteral("sample query"))));
   }
 
+  @Test
+  void query_expression() {
+    assertAnalyzeEqual(
+            dsl.query(
+                    dsl.namedArgument("query", DSL.literal("query_value"))),
```

Comment:
nit: change "query_value" to "sample query" or "field:query" or something similar.

Reply:
Fixed in https://github.com/Bit-Quill/opensearch-project-sql/pull/143/commits/ef2f4c9d6a4e671c82f08092b84ce9a7bf2cc3e9